I18n.md

```diff
@@ -30,7 +30,7 @@ const HelloWorld = ({t}) => (
  <div>{`Hi ${t('HELLO', { defaultValue: 'Hello world' })}`}</div>
 );
 
-export default translate(I18N_DOMAIN, { i18n: i18n.init() })(HelloWorld);
+export default translate(I18N_DOMAIN, { i18n: i18n.createInstance({}, () => {}) })(HelloWorld);
```

Comment:
Please externalize the only i18n default configuration.
So each translated component import it and set it instead of creating a default configuration on each component.
+ that will fix the linter error on line of code size